Library/Formula/vile.rb

```diff
@@ -0,0 +1,23 @@
+class Vile < Formula
+  desc "Vi Like Emacs"
+  homepage "http://invisible-island.net/vile/vile.html"
+  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
+  version "9.8"
+  sha256 "dea31c023aa7a90271056c712c826a5d16d19ad61df5cf5f299aa1025af280d6"
+
+  depends_on "reflex"
```

Comment:
That appears to have worked:

$ git push --force -u timbck2 adding_vile
Counting objects: 239, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (76/76), done.
Writing objects: 100% (239/239), 44.08 KiB | 0 bytes/s, done.
Total 239 (delta 187), reused 211 (delta 163)
To https://github.com/timbck2/homebrew.git
 + fe04b9b...2132753 adding_vile -> adding_vile (forced update)
Branch adding_vile set up to track remote branch adding_vile from timbck2.

Is there anything else I need to do? Thanks for all your collective help!

Tim Buck
Shared joy is a double joy; shared sorrow is half a sorrow. - Traditional
Swedish proverb

On Sun, Jan 17, 2016 at 2:21 PM, Mike McQuaid <notifications@github.com>
wrote:

> In Library/Formula/vile.rb
> <https://github.com/Homebrew/homebrew/pull/47698#discussion_r49952681>:
>
> > @@ -0,0 +1,23 @@
> > +class Vile < Formula
> > +  desc "Vi Like Emacs"
> > +  homepage "http://invisible-island.net/vile/vile.html"
> > +  url "http://invisible-island.net/datafiles/release/vile.tar.gz"
> > +  version "9.8"
> > +  sha256 "dea31c023aa7a90271056c712c826a5d16d19ad61df5cf5f299aa1025af280d6"
> > +
> > +  depends_on "reflex"
>
> You can git push --force
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/47698/files#r49952681>.
>